LanguageFeatures/Augmentation-libraries/augmentation_libraries_lib.dart

```diff
@@ -2,7 +2,7 @@
 // for details. All rights reserved. Use of this source code is governed by a
 // BSD-style license that can be found in the LICENSE file.
 
-/// @description Common library for augmentation libraries tests
+/// @description Common library for augmentations tests.
```

Comment:
I think this would flow more naturally:
```suggestion
/// @description Common library for augmentation tests.
```

Reply:
Fixed